tests/specs/mcd/verification.k

```diff
@@ -252,4 +251,9 @@ module VERIFICATION
     // (check-sat)
     rule chop(X *Int Y) /sWord chop(Y) => X requires #rangeUInt(256, X) andBool #rangeSInt(256, Y) andBool #rangeSInt(256, X *Int Y) andBool Y =/=Int 0
 
+    // ### dsvalue-peek-pass-rough
+
+    rule (B ==K false) ==K false => B
+    rule bool2Word((UINT8 ==K 0) ==K false) => UINT8 requires #rangeBool(UINT8)
```

Comment:
With lemmas from above comment, this can be replaced with lower-level lemmas:
```
rule notBool (X ==Int 0) => X ==Int 1 requires #rangeBool(X)
rule bool2Word(X ==Int 1)  => X         requires #rangeBool(X)

//Rcommended for symmetry, to avoid dealing with them next time
rule notBool (X ==Int 1) => X ==Int 0   requires #rangeBool(X)
```
And above rule should be made test instead.